exporter/newrelicexporter/transformer.go

```diff
@@ -28,19 +28,22 @@ import (
 )
 
 const (
-	unitAttrKey               = "unit"
-	descriptionAttrKey        = "description"
-	collectorNameKey          = "collector.name"
-	collectorVersionKey       = "collector.version"
-	instrumentationNameKey    = "instrumentation.name"
-	instrumentationVersionKey = "instrumentation.version"
-	statusCodeKey             = "otel.status_code"
-	statusDescriptionKey      = "otel.status_description"
-	spanKindKey               = "span.kind"
-	spanIDKey                 = "span.id"
-	traceIDKey                = "trace.id"
-	logSeverityTextKey        = "log.level"
-	logSeverityNumKey         = "log.levelNum"
+	unitAttrKey                     = "unit"
+	descriptionAttrKey              = "description"
+	collectorNameKey                = "collector.name"
+	collectorVersionKey             = "collector.version"
+	instrumentationNameKey          = "instrumentation.name"
+	instrumentationVersionKey       = "instrumentation.version"
+	droppedAttributesCountKey       = "otel.dropped_attributes_count"
+	droppedEventsCountKey           = "otel.dropped_events_count"
+	eventsDroppedAttributesCountKey = "otel.event.dropped_attributes_count"
```

Comment:
Why is this different from `droppedAttributesCountKey`?

Reply:
The main reason for this difference is to be more closely aligned with https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk_exporters/jaeger.md#events which uses that name for the dropped attributes counts on span events. Using the `otel.event` prefix did not seem like an appropriate thing to do when reporting the dropped attributes count for spans and logs.